docs/autoloop.md

```diff
@@ -0,0 +1,114 @@
+# Autoloop
```

Comment:
Would be nice to link to this document from the main README, otherwise people might not find it easily.

Reply:
yeah good idea!